tests/core/scene/a-scene.test.js

```diff
@@ -388,6 +388,70 @@ suite('a-scene (without renderer)', function () {
     });
   });
 
+  suite('resize', function () {
+    test('resize not in vr updates renderer size', function () {
+      var sceneEl = this.el;
+      AScene.prototype.resize.restore();
+      sceneEl.camera = { updateProjectionMatrix: function () {} };
+      sceneEl.canvas = document.createElement('canvas');
+      sceneEl.renderer = { setSize: function () {} };
```

Comment:
This code bellow is common to all the tests in the suite. Can we factor it out to a setup function? e.g: https://github.com/aframevr/aframe/blob/master/tests/core/scene/a-scene.test.js#L214

````javascript
 var sceneEl = this.el;
 AScene.prototype.resize.restore();
 sceneEl.camera = { updateProjectionMatrix: function () {} };
 sceneEl.canvas = document.createElement('canvas');
 sceneEl.renderer = { setSize: function () {} };
````

Reply:
Good idea, thank you